column/column.go

```diff
@@ -167,6 +167,15 @@ func (c *Col) GetTypeDesc() string {
 			elems[i] = strings.Replace(c.Elems[i], "'", "''", -1)
 		}
 		buf.WriteString(fmt.Sprintf("('%s')", strings.Join(elems, "','")))
+	case mysql.TypeFloat, mysql.TypeDouble:
```

Comment:
Seems `case mysql.TypeFloat, mysql.TypeDouble` is the same as default case, so is it needless?